extensions/mode-control-register.md

```diff
@@ -0,0 +1,53 @@
+# Overview
+
+This document describes the mode control register in a centralized location so that other extensions which use it can refer to a single document
+instead of duplicating it between all relevant extensions. The mode control register adds a new concept of _modes_ to the ISA. Modes are used to
+control things about the processor state which are binary-incompatible - that is, code written for a mode other than the mode the
+processor is actually in will almost certainly behave incorrectly.
+
+# Modes
+
+CR `1 0001`, equivalently `cr17` or "the MODE control register," holds a value indicating the current
+_mode_ of the processor. This has an initial value of `0`.
+
+## Base Mode (Real 16-bit Address Mode)
+
+The mode described by [the base isa](../../base-isa.md) is known as the
+Base mode. In that mode, pointers are 16 bits and there is no virtual memory, paging, or memory protection.
+Base mode is indicated by a value of 0 in `cr17`.
+
+## Real n-bit Address Mode
+
+In this mode, all addresses are treated as being `n` bits long. Addresses always refer to their sign extension to the highest supported address size.
+Entering this mode from a smaller n-bit address mode _must_ preserve the program counter. Returning to a smaller n-bit address mode must preserve the program
+counter _if possible_ - if the current program counter interpreted as the lower n-bit Address would not refer to the same location, then the behavior of
```

Comment:
There were two uses of "lower"... sometimes github's UI for indicating things like this is frustrating :(

Reply:
Looks like there are 3 and I missed one.